.gitignore

```diff
@@ -4,8 +4,6 @@
 sandbox/
 
 # Misc
-creds.json
```

Comment:
There is a high risk of uploading Master API keys removing this from the .gitignore

Reply:
Ok, my idea is that the user can now the structure of this file. I'm going to add a `creds.sample.json` instead 👍